django-drf-react-quickstart/mcbcapp/frontend/src/components/IntroPage.js

```diff
@@ -0,0 +1,48 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+//import logo from "../assets/icons/logo.png";
+
+//console.log(logo);
+
+class IntroPage extends Component{ 
+    render(){
+               
+            const backgroundStyle = {
+                backgroundColor: "#32669d", 
+                width: "1919.9px", 
+                height: "1080px"
+            };
+            const nameStyle1 = {
+                color: "white", 
+                fontSize:"50px",  
+                position: "center" 
+            };
+            const nameStyle2 = {
+                color: "white", 
+                fontSize:"20px",  
+                position: "center" 
+            };
+
+            return(
+                <body style={backgroundStyle}>
```

Comment:
use <div> instead of <body>, as i think it will already be surrounded by a <body> tag